addons/website_event_track/models/event_track.py

```diff
@@ -66,14 +66,14 @@ def _get_default_stage_id(self):
     date = fields.Datetime('Track Date')
     date_end = fields.Datetime('Track End Date', compute='_compute_end_date', store=True)
     duration = fields.Float('Duration', default=1.5, help="Track duration in hours.")
-    location_id = fields.Many2one('event.track.location', 'Room')
+    location_id = fields.Many2one('event.track.location', 'Location')
     event_id = fields.Many2one('event.event', 'Event', required=True)
     color = fields.Integer('Color', related="stage_id.color")
     priority = fields.Selection([
         ('0', 'Low'), ('1', 'Medium'),
         ('2', 'High'), ('3', 'Highest')],
         'Priority', required=True, default='1')
-    image = fields.Image("Image", related='partner_id.image_128', store=True, readonly=False)
+    image = fields.Image("Image")
```

Comment:
maybe a max width / max height ? I don't think we should keep too large images, what do you think ?